src/components/atoms/ChatMessage/ChatMessage.scss

```diff
@@ -1,82 +1,136 @@
 @import "scss/constants";
 
-$counterparty-text-border-radius: 28px 28px 28px 4px;
-$my-text-border-radius: 28px 28px 4px 28px;
+$counterparty-bottom-border-radius: 0 0 $spacing--xxl $spacing--xs;
+$counterparty-text-border-radius: $spacing--xxl $spacing--xxl $spacing--xxl
+  $spacing--xs;
+$my-text-bottom-border-radius: 0 0 $spacing--xs $spacing--xxl;
+$my-text-border-radius: $spacing--xxl $spacing--xxl $spacing--xs $spacing--xxl;
 
-.chat-message {
+$replies-max-height: 200px;
+
+$reply-icon-radius: 20px;
+
+.ChatMessage {
   align-self: flex-start;
   display: flex;
   align-items: flex-start;
   flex-direction: column;
-  margin-bottom: 16px;
+  margin-bottom: $spacing--lg;
 
-  &__text {
-    margin-bottom: 6px;
+  &:hover {
+    .ChatMessage__reply-icon {
+      display: flex;
+    }
+  }
+
+  &__bulb {
+    margin-bottom: $spacing--sm;
     width: auto;
     border-radius: $counterparty-text-border-radius;
     overflow-wrap: break-word;
     word-wrap: break-word;
     word-break: break-word;
     hyphens: auto;
-    background-color: $intermediate-grey;
-    padding: 12px 16px;
-    font-size: 0.9rem;
+    background-color: $secondary;
+
+    font-size: $font-size--md;
   }
 
-  &__info {
+  &__text-content {
+    position: relative;
     display: flex;
-    align-items: center;
-    font-size: 0.8rem;
+    flex-flow: column;
+    padding: $spacing--md $spacing--lg;
   }
 
-  &__author {
-    opacity: 0.8;
-    color: $white;
-    margin: 0 8px 0 4px;
+  &__text {
+    font-size: $font-size--md;
+  }
+
+  &__show-replies-button {
+    margin-top: $spacing--sm;
+    align-self: flex-start;
+  }
+
+  &__reply-icon {
+    display: none;
+    justify-content: center;
+    align-items: center;
+    z-index: z(chatmessage-reply-button);
+
+    position: absolute;
+
     cursor: pointer;
 
+    width: $reply-icon-radius;
+    height: $reply-icon-radius;
+
+    right: $spacing--sm;
+
+    bottom: calc(-#{$reply-icon-radius / 2});
+
+    border-radius: $border-radius--max;
+    background-color: $secondary--light;
+
     &:hover {
-      opacity: 1;
+      background-color: $secondary--lightest;
     }
   }
 
-  &__time {
-    opacity: 0.6;
-    font-weight: 300;
+  &__replies-content {
+    border-radius: $counterparty-bottom-border-radius;
+    overflow: hidden;
+    max-height: $replies-max-height;
   }
```

Comment:
the sole purpose of this wrapper is to hide overflow from `__replies` and to simulate the form of `__bulb`. Since I can't set `overflow: hidden` for `__bulb` because it would hide the `reply button`.

Reply:
Oh, actually I made it to prevent this behavior
![image](https://user-images.githubusercontent.com/63194656/118393999-b676b480-b64a-11eb-8d15-fbcaaa3f39b4.png)

Reply:
Fair enough. If it's needed, that's fine. Possibly add an explanation comment?

Reply:
@mike-lvov actually the reply button should be placed at the very bottom of the bubble (see the screenshot). I don't think we need an `overflow hidden` on the bubble (is the bubble the same thing as ` __bulb` ?), but just a` max-height` and an **overflow scroll-y** on the replies container 

<img width="506" alt="Capture d’écran 2021-05-17 à 14 26 42" src="https://user-images.githubusercontent.com/66247462/118488807-68ca7c80-b71c-11eb-8e6b-8c6c478f2345.png">

Reply:
Added in https://github.com/sparkletown/sparkle/pull/1374/commits/1c37309810e063a5f6a570371d22071088c1898d